src/SettingsMapper.php

```diff
@@ -63,16 +65,23 @@ public function save(
                     $payload = Crypto::encrypt($payload);
                 }
 
-                $config->getRepository()->updatePropertyPayload(
-                    $config->getGroup(),
-                    $name,
-                    $payload
+                return [
+                    'group' => $config->getGroup(),
+                    'name' => $name,
+                    'payload' => $payload,
+                ];
+            })
+            ->groupBy('group')
```

Comment:
Is this required, we're saving one settings class so we'll always have one group right?

Reply:
You're correct that there's no need to group, since it will always have one group.